sympy/integrals/tests/test_integrals.py

```diff
@@ -1475,3 +1475,10 @@ def test_issue_4311():
         (x**4/4 - 9*x**2/2, x <= -3),
         (-x**4/4 + 9*x**2/2 - S(81)/2, x <= 3),
         (x**4/4 - 9*x**2/2, True))
+
+def test_issue_15494():
+    x = symbols('x', real=True, positive=True)
```

Comment:
Is it necessary to set these assumptions?

Reply:
For this type x only it hangs.

Reply:
I'm worried about hanging. It should be found out why that would happen.

Reply:
ok, I am trying to find.